### PR TITLE
Tiny cleanup

### PR DESCRIPTION
**Pull request by pabzm**
This adds some comments and sorts the code a little. It also renames the app to 'dc-login-bot', and refactors checking the config for mandatory options to make it more concise.

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
-// Load config
-var config = require('rc')("login-dc", {
+// Load config from config-file.
+const appName = 'dc-login-bot'
```
Comment (Simon-Laux) [inline src/config.js:2]: so my old config file wont work anymore?
Also you might wanna add this file to the .gitgnore too, if you really want to rename it.

File: src/config.js [hunk 2]
```diff
@@ -9,19 +11,24 @@ var config = require('rc')("login-dc", {
     }
 })
 
-if (!config.email_address || !config.email_password) {
-    console.error("Missing configuration: Email address or password is missing.")
-    process.exit(1)
+const configFile = config.config || `./.${appName}rc`
```
Comment (Simon-Laux) [inline src/config.js:14]: what are you doing with this line?, I don't fully understand its existance

Comment (missytake) [reply src/config.js:14]: at least I think the config file name would be `~/.${appName}.rc`, not `./.${appName}rc`? so in ~, and with a . between appName and rc?

File: src/config.js (hunk 1 above)
Comment (missytake) [reply src/config.js:2]: we could probably rename the existing config file. But yeah, .gitignore makes sense.

File: src/config.js (hunk 2 above)
Comment (missytake) [reply src/config.js:14]: At least in production, the config file is in the home directory.

**Comment by pabzm** [comment]
This has been superseeded by other changes, sorry I didn't close it earlier!

**Pull request closed without merging**